img_styler/ui/common.py

```diff
@@ -196,6 +196,16 @@ async def update_processed_face(q: Q, save=False):
                                 step=0.1,
                                 value=q.client.prompt_strength,
                             ),
+                            ui.textbox(
+                                name="prompt_a",
+                                label="Added Prompt",
+                                value=q.client.prompt_a or "best quality, extremely detailed",
```

Comment:
As part of this, shall we also expose the guidance scale param?

Reply:
Sure. Will add those.